### PR TITLE
Fix usage of map_blocks

### PR DESCRIPTION
**Pull request by stephenworsley**
See https://github.com/SciTools/iris/pull/5767

### CODE REVIEW

File: esmf_regrid/schemes.py [hunk 1]
```diff
@@ -276,7 +275,7 @@ def _regrid_along_dims(regridder, data, dims, num_out_dims, mdtol):
     return result
 
 
-def _map_complete_blocks(src, func, dims, out_sizes):
+def _map_complete_blocks(src, func, active_dims, out_sizes, *args, **kwargs):
```
Comment (bjlittle) [inline esmf_regrid/schemes.py:278]: @stephenworsley Is your inclusion of `*args` here speculative of future `func` API?

At the moment `*args` doesn't seem to be used ... just curious.

**Review by bjlittle — CHANGES_REQUESTED** [review]
@stephenworsley Looks good 👍 

Just a simple question to service, also I guess existing testing covers this, right?

I wouldn't expect you to add tests to ensure that `partial` isn't used 😜

File: esmf_regrid/schemes.py (hunk 1 above)
Comment (stephenworsley) [reply esmf_regrid/schemes.py:278]: yeah, I suppose this is speculative for the time being, though the idea here is that I wanted to match the signature used by `map_blocks`. I decided to preserve both args and kwargs for the sake of completeness.

Comment (stephenworsley) [reply esmf_regrid/schemes.py:278]: This also matches the pattern I used in iris here https://github.com/SciTools/iris/pull/5767 which preserve args for the same reasons.

**Pull request merged**